app/presenters/comment_presenter.rb

```diff
@@ -19,11 +20,26 @@ def as_api_response
       author:           PersonPresenter.new(author).as_api_json,
       created_at:       created_at,
       mentioned_people: build_mentioned_people_json,
-      reported:         current_user.present? && reports.where(user: current_user).exists?
+      reported:         current_user.present? && reports.where(user: current_user).exists?,
+      interactions:     build_interactions_json
+    }
+  end
+
+  def  build_interactions_json
+    {
+      likes:            as_api(likes),
+      likes_count:      likes_count
     }
   end
 
   def build_mentioned_people_json
     mentioned_people.map {|m| PersonPresenter.new(m).as_api_json }
   end
+
+ def as_api(collection)
```

Comment:
- Layout/IndentationWidth: Use 2 (not 1) spaces for indentation.
- Layout/IndentationConsistency: Inconsistent indentation detected.